eventmesh-operator/config/samples/eventmesh_v1_connectors_rocketmq.yaml

```diff
@@ -86,11 +86,11 @@ spec:
       # resources describes the compute resource requirements and limits
       resources:
         requests:
-          memory: "512Mi"
-          cpu: "250m"
+          memory: 512Mi
+          cpu: 250m
         limits:
-          memory: "1024Mi"
-          cpu: "500m"
+          memory: 1024Mi
+          cpu: 500m
```

Comment:
The RocketMQ connector is a single-threaded application, which means it has limited consumption speed. It operates in scenarios where network I/O is dominant. In theory, it would require a maximum of 1 CPU when there is a sudden surge of messages from the upstream. I'm not sure if 0.5 CPU is sufficient. Could you please set the limit to 1 CPU or higher and use the "kubectl top pod" command to check the maximum CPU usage when the connector consumes messages?

Reply:
<img width="561" alt="image" src="https://github.com/apache/eventmesh/assets/91315508/acff8316-6bba-41d7-93c5-1cd3ffce2e9f">   

There may be some deviation. I have revised it according to your opinion.